chapters/en/slides/module2/module2_01.md

```diff
@@ -39,6 +39,8 @@ A few of the world's most prominent programmers of the time computing ballistic
 
 <img src='/module2/wires-computers-eniac-1946.png' width="50%" alt="A few of the world's most prominent programmers of the time computing ballistic trajectories on the ENIAC computer during the second world war."/>
 
+Unidentified U.S. Army photographer, Public domain, [via Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Reprogramming_ENIAC.png)
```

Comment:
Two women posed to show the "direct programming" of ENIAC done by setting hundreds of wires and thousands of switches, in the pre-Converter Code days (an instruction code written by Dr. John von Neumann, Dr. Richard Clippinger, Jean Jennings Bartik and Adele Goldstine and used for ENIAC programming 1947-1955). Standing: Ester Gerston and couching Gloria Gordon Bolotsky, both "Computers" for the Army during WWII and later second-generation ENIAC Programmers. "U.S. Army Photo" from the archives of the ARL Technical Library. 

Extracted from: https://ftp.arl.army.mil/ftp/historic-computers/

Reply:
Here you will find how should be cited https://ftp.arl.army.mil/ftp/historic-computers/#rights

Reply:
Thanks for finding that link! I don't think we need to insert the entire caption as part of citing the image. This is what it says at the bottom:

> All photos marked "U. S. Army Photo" are in the public domain, and may be used without fee, provided that each use is marked "U. S. Army Photo". 

I think we already do this in the current format, but I agree it could be nice to mention the original source even if it is not needed for public domain. I updated the citation and I will shoot them an email too since they said they appreciate knowing when this is used

Reply:
Ok, I guess if they agree with that caption should be fine.